system/modules/isotope/config/config.php

```diff
@@ -451,7 +451,10 @@
 $GLOBALS['ISO_NUM']["10,000.00"]    = array(2, '.', ",");
 $GLOBALS['ISO_NUM']["10.000,00"]    = array(2, ',', ".");
 $GLOBALS['ISO_NUM']["10'000.00"]    = array(2, '.', "'");
-
+$GLOBALS['ISO_NUM']["10000"] = [0, '.', ""];
```

Comment:
Have you tested this? I think I had an issue once with a definition like that due to it being interpreted as a numerical index.